msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/sdk/MsalSdk.java

```diff
@@ -101,11 +103,17 @@ public MsalAuthResult acquireTokenInteractive(@NonNull MsalAuthTestParams authTe
             );
         }
 
-
         final AcquireTokenParameters acquireTokenParameters = acquireTokenParametersBuilder.build();
 
         pca.acquireToken(acquireTokenParameters);
 
+        // Adding small sleep to make sure acquireToken is up before we start handling UI
```

Comment:
What happens otherwise?

Reply:
I think I did this in response to some cases having a UiSelector failure where it wouldn't be able to locate the username or password box.